src/index.ts

```diff
@@ -85,23 +101,79 @@ export default class BlueButton {
     };
   }
 
-  public generateAuthData(): AuthData {
+  async getExplanationOfBenefitData(
+    authToken: AuthorizationToken,
+    config: AxiosRequestConfig
+  ) {
+    return await getFhirResource(
+      FhirResourceType.ExplanationOfBenefit,
+      authToken,
+      this,
+      config
+    );
+  }
+
+  async getPatientData(
+    authToken: AuthorizationToken,
+    config: AxiosRequestConfig
+  ) {
+    return await getFhirResource(
+      FhirResourceType.Patient,
+      authToken,
+      this,
```

Comment:
in the scope of an async method in a class, "this" is not a reference to the class instance - that's what I ran into, searched internet and not finding anything though....